### PR TITLE
EKA-MA-1, EKA-MA-2 | fixing HIU purpose of code being ENUM, instead of string

### PR DESCRIPTION
**Pull request by angshu**

### CODE REVIEW

File: src/main/java/in/org/projecteka/hiu/consent/ConceptValidator.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        ConcurrentHashMap<String, Map<String, String>> vsCodeMap = readValueSetFromResource();
+        purposesOfUse = vsCodeMap.get(HEALTH_INFORMATION_PURPOSE_OF_USE);
+        hiTypes = vsCodeMap.get(HEALTH_INFORMATION_TYPE);
+    }
+
+
+    private ConcurrentHashMap<String, Map<String, String>> readValueSetFromResource() throws IOException {
+        try (InputStream in = valueSetsResource.getInputStream()) {
+            ByteArrayOutputStream result = new ByteArrayOutputStream();
+            byte[] buffer = new byte[1024];
```
Comment (ganesanarun) [inline src/main/java/in/org/projecteka/hiu/consent/ConceptValidator.java:42]: Would not it be awesome to make it more obvious by defining class with properties which are required to read value set 🤔 and easier for new person who's going to onboard to understand?

Comment (ganesanarun) [reply src/main/java/in/org/projecteka/hiu/consent/ConceptValidator.java:42]: Except this and some extra empty spaces it's looking good.

Comment (angshu) [reply src/main/java/in/org/projecteka/hiu/consent/ConceptValidator.java:42]: as in different valuesets for purpose and HIType? Thought about it, but then for simplicity sake, kept in a single json. UI also uses the same.

Comment (angshu) [reply src/main/java/in/org/projecteka/hiu/consent/ConceptValidator.java:42]: Maybe you meant different validators for different concepts? Thought about that too. Then thought these would be the same. Conceptually, they are just validating same type of concepts, just from different system/resource perspective.

**Review by ganesanarun — APPROVED** [review]

**Pull request merged**